tests/monitoring/monitoring.go

```diff
@@ -190,6 +164,57 @@ var _ = Describe("[sig-monitoring]Monitoring", Serial, decorators.SigMonitoring,
 
 })
 
+func disableVirtHandler(virtClient kubecli.KubevirtClient, originalKv *v1.KubeVirt) error {
+	customizedComponents := v1.CustomizeComponents{
+		Patches: []v1.CustomizeComponentsPatch{
+			{
+				ResourceName: virtHandler.deploymentName,
+				ResourceType: "DaemonSet",
+				Patch:        `{"spec":{"template":{"spec":{"nodeSelector":{"kubernetes.io/hostname":"does-not-exist"}}}}}`,
+				Type:         v1.StrategicMergePatchType,
+			},
+		},
+	}
+
+	patchBytes, err := patch.New(patch.WithAdd("/spec/customizeComponents", customizedComponents)).GeneratePayload()
+	if err != nil {
+		return err
+	}
+
+	_, err = virtClient.KubeVirt(originalKv.Namespace).Patch(context.Background(), originalKv.Name, types.JSONPatchType, patchBytes, metav1.PatchOptions{})
+	return err
+}
+
+func restoreVirtHandler(virtClient kubecli.KubevirtClient, originalKv *v1.KubeVirt) error {
+	patchBytes, err := patch.New(patch.WithRemove("/spec/customizeComponents")).GeneratePayload()
+	if err != nil {
+		return err
+	}
+
+	_, err = virtClient.KubeVirt(originalKv.Namespace).Patch(context.Background(), originalKv.Name, types.JSONPatchType, patchBytes, metav1.PatchOptions{})
+	return err
+}
+
+func waitForVirtHandlerNodeSelector(offset int, virtClient kubecli.KubevirtClient, namespace, expectedValue string, timeout, polling time.Duration) {
```

Comment:
I think the offset may not be a variable, but I would not block for this
/lgtm

Reply:
just want to make it as reusable as possible but don't have strong opinion on it. I can change if you prefer.
let me know

Reply:
all good